profile/profile.json

```diff
@@ -0,0 +1,83 @@
+{
+  "education": [
+    {
+      "awards_achievements": [
+        "GPA: 3.9/4.5"
+      ],
+      "degree": "Software Engineering Technology",
+      "department": "Engineering Technology and Applied Science",
+      "duration": "2021-2023",
+      "school_name": "Centennial College"
+    },
+    {
+      "awards_achievements": null,
+      "degree": "Bachelor of Science in Architecture",
+      "department": "Engineering and Architecture",
+      "duration": "2013-2019",
+      "school_name": "University of the Assumption"
+    }
+  ],
+  "github_repo_root_url": "https://github.com/jarviscanada/jarvis_data_eng_IvanZapanta",
+  "highlighted_projects": [
+    {
+      "description": "Created a sophisticated Patient Health Tracker System by utilizing the power of React and JavaScript for a feature-rich frontend. The application's frontend communicates smoothly with the backend through GraphQL, optimizing data exchanges. Patient health data is securely stored and retrieved using MongoDB, ensuring the system's reliability and scalability. Overall, the project generated an efficient healthcare data management system, ultimately contributing to improved patient care and heightened healthcare service efficiency.",
```

Comment:
Try to quantify the effect if possible. For instance, the last line states that this project "improved patient care and heightened healthcare service efficiency". Are you able to give a rough estimate of how much it improved it?